docs/howto/turn-a-hooks-based-charm-into-an-ops-charm.md

```diff
@@ -206,7 +205,7 @@ That allows us to fetch the Relation wherever we need it and access its contents
         )
 ```
 
-Note how `relation.data` provides an interface to the relation databag (more on that [here](https://juju.is/docs/sdk/relations#heading--relation-data)) and we need to select which part of that bag to access by passing an `ops.model.Unit` instance.
+Note how `relation.data` provides an interface to the relation databag (see [](ops.Relation.data)) and we need to select which part of that bag to access by passing an `ops.model.Unit` instance.
```

Comment:
This was going to [a Juju reference page previously](https://web.archive.org/web/20221102151915/https://juju.is/docs/sdk/integration#heading--integration-data) rather than to the ops API reference. I can't figure out where that is now: @tmihoc did it get removed?

If it doesn't exist any more, I think maybe linking to the section in the relation how-to might be better here?